.travis.yml

```
@@ -1,17 +1,14 @@
 language: php
 
 php:
-  - 7.1
-  - 7.2
-  - 7.3
-  - 7.4
+  - 8.0
 
 before_script:
   - composer update
 
 script:
   - ./vendor/bin/phpunit --disallow-test-output --coverage-clover ./clover.xml
```

```suggestion
  - XDEBUG_MODE=coverage ./vendor/bin/phpunit --disallow-test-output --coverage-clover ./clover.xml
```
Maybe try this? Sorry, this is new to me too as I've not had a chance to play with Xdebug 3 yet :)

---

Sam story here. But together, I think we can find the solution and learn from it ;)